public/static/docs/command-reference/update.md

```diff
@@ -75,3 +73,14 @@ stable.
 > Note that `dvc update` updates the `rev_lock` field of the corresponding
 > [DVC-file](/doc/user-guide/dvc-file-format) (when there are changes to bring
 > in).
+
+To update from and also lock to a specific revision of a <abbr>data
+artifact</abbr>, we may use the `--rev` option this time:
+
+```dvc
+$ dvc update --rev v2 model.pkl.dvc
+Importing 'model.pkl (git@github.com:iterative/example-get-started)'
+-> 'model.pkl'
+```
+
+Since the source files has changed, we see the results of our "update".
```

Comment:
```suggestion
Since the source files are different at `v2` comparing to the currently locked revision, we see the side side effects of our "update" as different results.
```

?

Reply:
Maybe something like
```suggestion
This actually has an effect because the `model.pkl.dvc` file at tag `v2` points to different data that what's currently in the `rev_lock` field of the import stage.
```
Notice I also avoided the term "lock" here as well.
(Please rewrap.)

Reply:
I would also keep the entire

> Note that `dvc update` updates the `rev_lock` field...

note here, at the end of the doc.